yarn-project/noir-libs/noir-aztec/src/state_vars/set.nr

```diff
@@ -64,13 +66,20 @@ impl<Note, N> Set<Note, N> {
         destroy_note(context, self.storage_slot, note, self.note_interface);
     }
 
-    fn get_notes<S, P>(
+    fn get_notes<FILTER_ARGS>(
```

Comment:
We should probably add a comment to say where to read more info about `FILTER_ARGS` - perhaps a link to the NoteGetterOptions file and a link to the docs?
I'm just thinking, if someone reads this `Set` method without having seen examples, they'll not know how to construct this template argument without help.